.github/workflows/release.yaml

```diff
@@ -259,15 +260,14 @@ jobs:
     name: Release
     runs-on: ubuntu-latest
     needs:
-      - lint
       - macos
       - windows
       - linux
       - linux-cross
       - musllinux
       - musllinux-cross
       - pypy
-    if: "startsWith(github.ref, 'refs/tags/')"
+    if: "github.ref == 'refs/heads/main' && startsWith(github.ref, 'refs/tags/')"
```

Comment:
Can this condition be true? If `github.ref` is exactly `'refs/heads/main'`, then how could it start with `'refs/tags/'`?

Reply:
Done!